### PR TITLE
fix: variable ad height on articlie list pages

### PR DESCRIPTION
**Pull request by colincclark**
https://nidigitalsolutions.jira.com/browse/REPLAT-3836

This should not adversely affect ads on any of the pages as the height is just set to be "auto", although this was difficult to test as ads seems to be very flakey. It shows intermittently and on some stories, not at all

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/56f894f348c7628fbff301dad6b8cdfacafde516/index.html

File: packages/ad/src/dom-context-prop-types.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import PropTypes from "prop-types";
 
 export const propTypes = {
   data: PropTypes.shape({}),
-  height: PropTypes.number.isRequired,
+  height: PropTypes.number,
```
Comment (tuncaulubilge) [inline packages/ad/src/dom-context-prop-types.js]: It might be worth splitting this for native, as it's still required for native

Comment (colincclark) [reply packages/ad/src/dom-context-prop-types.js]: done

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/66ff5315c85abefb618be7aad7da193d9d1201e5/index.html

**Pull request merged**